### ISSUE TITLE
Privacy Center allows for both email and phone number to be used in current state. 

### ISSUE THREAD

**Issue by nicolas-ethyca** [ethyca/fides#2538]
### Bug Description

Privacy Center allows for email and phone to be used in current state. Currently, it is an XOR. 

#### Steps to Reproduce

1. Open Privacy Center
2. Add a phone
3. Add an email
4. Submit request

### Expected behavior

It disables one field of the two fields to be xor 

### Screenshots

_If applicable, add screenshots to help explain your problem._

### Environment

* Version: 2.6.3
* OS: OSX

**Comment by Roger-Ethyca** [comment]
1. bring up test_env
2. click Access your data 
3. add both email and phone 
4. log into the admin ui 
5. add a connector (i used shopify) 
6. approve the request 
7. the request will error 
8. go to view details 
9. and view the last request and you will see the error 

error:  Only one identity can be specified for SaaS connector traversal
![Screenshot 2023-03-08 at 4 44 02 PM](https://user-images.githubusercontent.com/115798183/223857306-60679cf9-bc9a-4e21-ba49-f07fcc55c80b.png)

**Comment by galvana** [comment]
I don't think this ticket addresses the root problem. What I think we should do is accept both email and phone number identities in the privacy center but make sure the privacy request isn't verified until both identities are verified (email verification and SMS verification).

**Comment by Roger-Ethyca** [comment]
moving to done 
tested with 
root_user - accessible - passed
owner -   accessible - passed 
all other it is not accessible
